contrib/arch/d1x-rebirth/PKGBUILD

```diff
@@ -1,36 +1,44 @@
-# Contributor : zico <zico@dxx-rebirth.com>
+# Contributor: zico <zico@dxx-rebirth.com>
+# Contributor: Dominic Radermacher <dominic.radermacher@gmail.com>
+# Maintainer: SanskritFritz (gmail)
 
 pkgname=d1x-rebirth
-pkgver=0.61
-pkgrel=1
-pkgdesc="An enhanced engine to play with Descent1 datas"
-arch=('i686' 'x86_64')
+pkgver=20191121
+pkgrel=2
+pkgdesc="An enhanced engine to play with Descent1 data."
+arch=('x86_64')
```

Comment:
Rebirth still supports i686 targets.  What would be required to leave this enabled?

Reply:
Archlinux doesn't support i686 anymore so we drop that target most of the time. But I can leave it enabled, it causes no problems but I thought I'd stick to archlinux recommendations.